Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -295,6 +295,22 @@ open class Kommunicate: NSObject, Localizable {
         navVC.modalPresentationStyle = .fullScreen
         viewController.present(navVC, animated: true, completion: nil)
     }
+    
+    /**
+      Launch chat list from a ViewController.
```

Comment:
Update the description so that it's clear that this function is for launching the screen in a container view.

Reply:
done